src/teleop/jetson/jetson_teleop.py

```diff
@@ -201,6 +202,11 @@ def ra_open_loop_control(self, msg: Joy) -> None:
             self.ra_config["finger"]["multiplier"] * self.filter_xbox_button(msg.buttons, "y", "a"),
             self.ra_config["gripper"]["multiplier"] * self.filter_xbox_button(msg.buttons, "b", "x"),
         ]
+        if self.ra_slow_mode == True:
+            for i, (key, value) in enumerate(self.ra_config.items()):
```

Comment:
I think you should just use `RA_NAMES`. You don't need the values of the `ra_config` dictionary, but also I don't think you can be sure the order is the same as you expect, which is important since you're lining up the values with indices.